public/video-ui/src/components/ContentChangeDetails/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ManagedField, ManagedForm } from '../ManagedForm';
+import TextInput from '../FormFields/TextInput';
+
+class ContentChangeDetails extends React.Component {
+  static propTypes = {
+    video: PropTypes.object.isRequired
+  };
+
+  getCreatedByField = () => {
+    return (
+      <ManagedField
+        fieldLocation="contentChangeDetails.created.user.email"
+        name="Created by"
+        disabled={true}
+      >
+        <TextInput />
+      </ManagedField>
+    );
+  };
+
+  getModifiedByField = () => {
+    return (
+      <ManagedField
+        fieldLocation="contentChangeDetails.lastModified.user.email"
+        name="Last modified by"
+        disabled={true}
+      >
+        <TextInput />
+      </ManagedField>
+    );
+  };
+
+  render() {
+    const { video } = this.props;
+
+    if (!video || !video.id) {
```

Comment:
`video` is marked as required in the props. Would we ever get a video without an id?

Reply:
This is for atom creation, as the component is shared. An incoming PR will tidy this up a bit, pushing this further up the tree for a cleaner and clearer design.